codegen/table_test.go

```diff
@@ -31,8 +31,12 @@ type (
 			IntCol    int    `json:"int_col,omitempty"`
 			StringCol string `json:"string_col,omitempty"`
 		}
-		IntArrayCol    []int      `json:"int_array_col,omitempty"`
-		StringArrayCol []string   `json:"string_array_col,omitempty"`
+		IntArrayCol    []int  `json:"int_array_col,omitempty"`
+		IntArrayColToo []*int `json:"int_array_col_too,omitempty"`
```

Comment:
```suggestion
		IntPointerArrayCol []*int `json:"int_pointer_array_col,omitempty"`
```

[non blocking] Since the convention is to use the type of the field, maybe keep that? See `TimePointerCol *time.Time` a few lines below.

Also might be better to use a name that describes the field, and not what's it's meant to be transformed to